bucket/shutter.json

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://www.den4b.com/products/shutter",
+    "version": "4.4",
+    "license": {
+        "identifier": "CC-BY-NC-ND-3.0",
+        "url": "https://www.den4b.com/license"
+    },
+    "description": "A multifunctional scheduling utility, which has a user friendly and easy-to-use interface and supports many different Events and Actions.",
+    "url": "https://www.den4b.com/download/shutter/portable/4.4?token=scoop#/dl.zip",
+    "hash": "a2cc77e2ecbc126398723aa57dfe00e653747d0997b01cc0d7ebb04f78162fb3",
+    "bin": "Shutter.exe",
+    "shortcuts": [
+        [
+            "Shutter.exe",
+            "Shutter"
+        ]
+    ],
+    "pre_install": "if (!(Test-Path \"$persist_dir\\Settings.ini\")) { New-Item -Path \"$dir\\Settings.ini\" -ItemType 'File' | Out-Null }",
```

Comment:
- Installers are always before binaries and shortcuts
- Do not use `-ItemType` for files and `-Path` in general for basic cmdlets

Reply:
Thanks!